java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/db/DBAddConnection.java

```diff
@@ -148,8 +158,9 @@ public CompletableFuture<Object> processCommand(NbCodeLanguageClient client, Str
                     if (!selectedItems.isEmpty()) {
                         int i = ((Double) selectedItems.get(0).getUserData()).intValue();
                         JDBCDriver driver = drivers[i];
+                        String urlTemplate = urlTemplates.get(driver.getClassName());
```

Comment:
give a safe fallback in unlikely case the driver classname will not be registered in the Map.

Reply:
fixed in 4b935e8